systers_portal/meetup/views.py

```diff
@@ -12,3 +12,17 @@ class MeetupLocationAboutView(MeetupLocationMixin, TemplateView):
 
     def get_meetup_location(self):
         return get_object_or_404(MeetupLocation, slug=self.kwargs['slug'])
+
+
+class MeetupView(MeetupLocationMixin, TemplateView):
```

Comment:
get_context_data is for consistency that is necessary for about_button.html snippet, and for rest other snippet also(whenever we will add links to them) we will need some common access to slug from all the views.